modules/openvpn/ecr.tf

```diff
@@ -0,0 +1,8 @@
+resource "aws_ecr_repository" "openvpn" {
```

Comment:
if we are going to use a public ECR, we don't need this, and we could immediately refer to our own repo in the task definition